docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/configuration/rs-rdma.md

```diff
@@ -0,0 +1,108 @@
+---
+id: rs-rdma
+title: Enable RDMA for Volume Targets
+keywords:
+ - OpenEBS Replicated PV Mayastor
+ - Replicated PV Mayastor Configuration
+ - Configuration
+ - RDMA
+ - Enable RDMA
+ - Enable RDMA for OpenEBS Replicated PV Mayastor
+description: This guide will help you to enable RDMA for OpenEBS Replicated PV Mayastor.
+---
+
+# Enable RDMA for Volume Targets
+
+The RDMA feature in Replicated PV Mayastor enables you to achieve better throughput and latency outcomes for the applications using Replicated PV Mayastor Volumes. This feature provides a significant performance boost by leveraging RDMA technology. A kubernetes cluster consisting of nodes that have RNIC (RDMA Network Interface Card) can make use of this feature.
+
+## Prerequisites
+
+Before enabling and using RDMA in Replicated PV Mayastor, ensure the following prerequisites are met:
+
+1. **Interface Validation**
+
+  Ensure that the interface name specified in `io-engine.target.nvmf.iface` exists on all io-engine nodes. Misconfiguration of this parameter can lead to unexpected behavior.
+  If the interface name is invalid or not RDMA-capable on a node, Replicated PV Mayastor will run with only default tcp support for such nodes.
+
+2. **Application Node Requirements**
+
+  Application nodes must also be RDMA-capable to connect to RDMA-enabled volume targets. This capability is independent of the Helm-provided interface name for `io-engine.target.nvmf.iface`. RDMA connections will work as long as the application node has an RDMA-capable device.
+
+3. **TCP Fallback Behavior**
+
+  If an application is hosted on a non-RDMA-capable node, connections to RDMA-enabled volume targets will default to TCP. To prevent this fallback behavior, set the Helm chart value `csi.node.nvme.tcpFallback` to `false`. In this configuration, the initiator (application) node will continuously attempt RDMA connections, resulting in connection failures. To mitigate this, you can either keep the fallback option enabled or relocate the application pod to an RDMA-capable node, provided the pod's resource type allows for such movement.
+
+4. **Soft-RoCEv2 Support**
+
+  RDMA devices can be manually created using `ibverbs` utilities. For example, to create an RDMA device named `rxe0` on an Ethernet interface `eth0`, run:
```

Comment:
Need to explain the need for this. Something on the lines "Mayastor also supports software RDMA if the node does not have RDMA-capable hardware network interface cards."